obspy/core/tests/test_waveform_plugins.py

```diff
@@ -174,7 +178,10 @@ def test_isFormat(self):
                     filelist.extend([os.path.join(directory, _i) for _i in
                                      files])
                 for file in filelist:
+                    if any([n in file for n in KNOWN_FALSE]):
+                        continue
                     if isFormat(file) is True:  # pragma: no cover
+                        print(file)
```

Comment:
Do we need to print it? Doesn't it go into `false_positives` and print from there?